scanmem.c

```diff
@@ -67,6 +67,7 @@ globals_t sm_globals = {
         REGION_HEAP_STACK_EXECUTABLE_BSS, /* region_detail_level */ 
         1,                      /* dump_with_ascii */
         0,                      /* reverse_endianness */
+        0,                      /* no_pthread */
```

Comment:
The comment is wrong here, should be `no_ptrace`.

Reply:
> The comment is wrong here, should be `no_ptrace`.

Done.  Rebased to 76b24ca2cae5748d7dd34f1eac30e19b632b3142.

Reply:
Done. Rebased to 76b24ca.

Reply:
Done. Rebased to 76b24ca.